docs/component-dispatcher.md

```diff
@@ -0,0 +1,49 @@
+# Dispatcher Service
+
+The dispatcher service is a central component in the hub and spoke service collaboration. 
+It is responsible for delivering messages from one application to one or more target applications based on a tenant specific configuration.
+
+![Dispatcher Service Overview](https://github.com/openintegrationhub/openintegrationhub/blob/dispatcher/Assets/component_dispatcher_0.1.png)
+
+## Service
+### Entry Point
+
+The service is triggered by messages arriving through a SDF Adapter Component used in a Connector Flow. 
+
```

Comment:
Is the data really received by sdf adapter? 
Is it possible that the dispatcher component receives data from another service? (E.g. data was validated by some sort of validation service and is then passed to the dispatcher component)